src/DynamoCore/Core/NodeModelAssemblyLoader.cs

```diff
@@ -199,6 +199,44 @@ internal static bool ContainsNodeModelSubType(Assembly assem)
             return assem.GetTypes().Any(IsNodeSubType);
         }
 
+        internal static bool ContainsNodeModelSubTypeReflectionLoaded(Assembly assem, Type nodeModelType = null)
+        {
+            if(nodeModelType is null)
+                return assem.GetTypes().Any(x => IsNodeSubType(x));
+            
+            return assem.GetTypes().Any(x => IsNodeSubTypeReflectionLoaded(x, nodeModelType));
+        }
+
+        internal static bool IsNodeSubTypeReflectionLoaded(Type t, Type nodeModelType)
```

Comment:
I think this deserves a summary ///

Reply:
also, this function is not defined anywhere else already?

Reply:
hm, dont think it is. I haven't come across it at least, anywhere specific you think it might be?